nilearn/_utils/testing.py

```diff
@@ -10,6 +10,8 @@
 import sys
 import tempfile
 import warnings
+import resource
```

Comment:
It is for Unix systems. I probably should skip tests if not available / setrlimit or getrlimit raise a resource.error.